include/itkNPasteImageFilter.h

```diff
@@ -0,0 +1,218 @@
+  using Superclass = InPlaceImageFilter<TInputImage, TOutputImage>;
+  using Pointer = SmartPointer<Self>;
+  using ConstPointer = SmartPointer<const Self>;
+
+  /** Method for creation through the object factory. */
+  itkNewMacro(Self);
+
+  /** Run-time type information (and related methods). */
+  itkTypeMacro(NPasteImageFilter, InPlaceImageFilter);
+
+  /** Typedefs from Superclass */
+  using InputImagePointer = typename Superclass::InputImagePointer;
+  using OutputImagePointer = typename Superclass::OutputImagePointer;
+
+  /** Typedef to describe the output and input image region types. */
+  using InputImageType = TInputImage;
+  using OutputImageType = TOutputImage;
+  using SourceImageType = TSourceImage;
+  using OutputImageRegionType = typename OutputImageType::RegionType;
+  using InputImageRegionType = typename InputImageType::RegionType;
+  using SourceImageRegionType = typename SourceImageType::RegionType;
+
+  using SourceImagePointer = typename SourceImageType::Pointer;
+  using SourceImageConstPointer = typename SourceImageType::ConstPointer;
+
+  /** Typedef to describe the type of pixel. */
+  using OutputImagePixelType = typename OutputImageType::PixelType;
+  using InputImagePixelType = typename InputImageType::PixelType;
+  using SourceImagePixelType = typename SourceImageType::PixelType;
+
+  /** Typedef to describe the output and input image index and size types. */
+  using OutputImageIndexType = typename OutputImageType::IndexType;
+  using OutputImageSizeType = typename OutputImageType::SizeType;
+  using InputImageIndexType = typename InputImageType::IndexType;
+  using InputImageSizeType = typename InputImageType::SizeType;
+  using SourceImageIndexType = typename SourceImageType::IndexType;
+  using SourceImageSizeType = typename SourceImageType::SizeType;
+  using DecoratedSourceImagePixelType = SimpleDataObjectDecorator<SourceImagePixelType>;
+
+  using InputSkipAxesArrayType = FixedArray<bool, InputImageType::ImageDimension>;
+
+
+  /** ImageDimension enumeration */
+  static constexpr unsigned int InputImageDimension = InputImageType::ImageDimension;
+  static constexpr unsigned int OutputImageDimension = OutputImageType::ImageDimension;
+  static constexpr unsigned int SourceImageDimension = SourceImageType::ImageDimension;
+
+  /** Set/Get the destination index (where in the first input the second
+   * input will be pasted. */
+  itkSetMacro(DestinationIndex, InputImageIndexType);
+  itkGetConstMacro(DestinationIndex, InputImageIndexType);
+
+  /** Set/Get the an array describing which axes in the destination image to skip
+   *
+   * The axes with true values are set to 1, to fill the difference between the dimension of the input and source image.
+   * The number of true value in DestinationSkipAxes plus the DestinationImageDimension must equal the
+   * InputImageDimension.
+   *
+   * By default this array contains SourceImageDimension false values followed filled with values for the remainder.
```

Comment:
By default this array contains SourceImageDimension false values followed filled with values for the remainder. ->
By default this array contains SourceImageDimension false values followed by true values for the remainder.